hack/run-conformance.sh

```diff
@@ -0,0 +1,54 @@
+set -euo pipefail
+
+## Create kind cluster.
+KIND_NODE_IMAGE=${KIND_NODE_IMAGE:-"docker.io/kindest/node:v1.24.0@sha256:0866296e693efe1fed79d5e6c7af8df71fc73ae45e3679af05342239cdc5bc8e"}
+kind create cluster \
+    --name envoy-gateway-conformance \
+    --image "${KIND_NODE_IMAGE}"
+
+
+## Install metallb.
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/namespace.yaml
+if ! kubectl get secret -n metallb-system memberlist; then
+    kubectl create secret generic -n metallb-system memberlist --from-literal=secretkey="$(openssl rand -base64 128)"
+fi
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.12.1/manifests/metallb.yaml
+# Apply config with addresses based on docker network IPAM
+subnet=$(docker network inspect kind | jq -r '.[].IPAM.Config[].Subnet | select(contains(":") | not)')
+# Assume default kind network subnet prefix of 16, and choose addresses in that range.
+address_first_octets=$(echo ${subnet} | awk -F. '{printf "%s.%s",$1,$2}')
+address_range="${address_first_octets}.255.200-${address_first_octets}.255.250"
+kubectl apply -f - <<EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: default
+      protocol: layer2
+      addresses:
+      - ${address_range}
+EOF
+
+## Install Gateway API CRDs and webhook.
+GATEWAY_API_VERSION=$(grep "sigs.k8s.io/gateway-api" go.mod | awk '{print $2}')
```

Comment:
Use `go list -m sigs.k8s.io/gateway-api` instead of trying to parse `go.mod` with awk.